packages/terra-action-footer/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Removed "View Component Source Code" from docs in favour of source code badge
 ### Added
```

Comment:
nit: For all the changlogs add a new line between the changed and added section.